packages/shared/src/modules/activity-logs/libs/validation-schemas/libs/constants/email-regex.constant.ts

```diff
@@ -0,0 +1,4 @@
+const EMAIL_REGEX =
```

Comment:
We need to name it more specifically so that we understand how it differs from built-in email validation. Does it match emails with github or what?

Reply:
something like GIT_EMAIL_REGEX